### PR TITLE
code and doc maintenance

### LINKED ISSUE #130: notes
- `attr(x, 'observation')` is used instead of `attr(x, 'observations')` in cp plot and ap
- `size=2` is not used in categorical cp plot but remains as default function value

### LINKED ISSUE #131: CRAN NOTE
It looks like URLs must end with `/`

```
  Found the following (possibly) invalid URLs:
    URL: https://pbiecek.github.io/ema (moved to https://pbiecek.github.io/ema/)
      From: man/accumulated_dependence.Rd
            man/aggregate_profiles.Rd
            man/calculate_oscillations.Rd
            man/calculate_variable_profile.Rd
            man/ceteris_paribus.Rd
            man/cluster_profiles.Rd
            man/conditional_dependence.Rd
            man/feature_importance.Rd
            man/partial_dependence.Rd
            man/plot.aggregated_profiles_explainer.Rd
            man/plot.ceteris_paribus_2d_explainer.Rd
            man/plot.ceteris_paribus_explainer.Rd
            man/plot.ceteris_paribus_oscillations.Rd
            man/plot.feature_importance_explainer.Rd
            man/plotD3_aggregated_profiles.Rd
            man/print.feature_importance_explainer.Rd
```

### LINKED ISSUE #134: N = NULL in pdp/ale/cd returns error
There probably should be `if (!is.null(N))` like in [`feature_importance`](https://github.com/ModelOriented/ingredients/blob/91c9d39e2676b7517bbea699e78319567be6c961/R/feature_importance.R#L206)

https://github.com/ModelOriented/ingredients/blob/91c9d39e2676b7517bbea699e78319567be6c961/R/accumulated_dependence.R#L108

### PR DESCRIPTION
**Pull request by hbaniecki**
closes #130 closes #131 fixes #134 
aut -> ctb 
`randomForest` -> `ranger` 
`titanic` -> `titanic_imputed`

### CODE REVIEW

**Comment by hbaniecki** [comment]
fixes #134

File: R/ceteris_paribus.R [hunk 1]
```diff
@@ -41,23 +41,21 @@
...
-#' library("randomForest")
-#' model_titanic_rf <- randomForest(survived ~.,  data = titanic_imputed)
+#' library("ranger")
+#' model_titanic_rf <- ranger(survived ~., data = titanic_imputed, probability = TRUE)
 #'
 #'
 #' explain_titanic_rf <- explain(model_titanic_rf,
 #'                               data = titanic_imputed[,-8],
 #'                               y = titanic_imputed[,8],
-#'                               label = "Random Forest v7",
```
Comment (pbiecek) [inline R/ceteris_paribus.R:60]: why this label is removed? it can be changed to Ranger v7

File: R/cluster_profiles.R [hunk 2]
```diff
@@ -36,14 +37,13 @@
...
-#' library("randomForest")
-#' model_titanic_rf <- randomForest(survived ~.,  data = titanic_imputed)
-#' model_titanic_rf
+#' library("ranger")
+#' model_titanic_rf <- ranger(survived ~., data = titanic_imputed, probability = TRUE)
 #'
 #' explain_titanic_rf <- explain(model_titanic_rf,
 #'                               data = titanic_imputed[,-8],
 #'                               y = titanic_imputed[,8],
-#'                               label = "Random Forest v7")
```
Comment (pbiecek) [inline R/cluster_profiles.R:46]: as above, let's keep the label

File: vignettes/vignette_titanic.Rmd [hunk 3]
```diff
@@ -21,11 +21,11 @@ knitr::opts_chunk$set(
 # Data for Titanic survival
 
 Let's see an example for `DALEX` package for classification models for the survival problem for Titanic dataset.
-Here we are using a dataset `titanic` avaliable in the `DALEX` package. Note that this data was copied from the `stablelearner` package.
+Here we are using a dataset `titanic_imputed` avaliable in the `DALEX` package.
```
Comment (pbiecek) [inline vignettes/vignette_titanic.Rmd]: let's keep here an information about the source (stablelearner) of the data.

**Review by pbiecek — COMMENTED** [review]
Thanks a lot, great job!
what about these three small changes

File: R/ceteris_paribus.R (hunk 1 above)
Comment (hbaniecki) [reply R/ceteris_paribus.R:60]: I hope that `"ranger forest"` is ok. (edited all examples)

File: vignettes/vignette_titanic.Rmd (hunk 3 above)
Comment (hbaniecki) [reply vignettes/vignette_titanic.Rmd]: Information kept.

**Review by pbiecek — APPROVED** [review]
ok, thanks

**Pull request merged**